hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/UsingNuma.md

```diff
@@ -0,0 +1,180 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# NUMA
+
+Non-uniform memory access (NUMA) is a computer memory design used in multiprocessing,
+where the memory access time depends on the memory location relative to the processor.
+Under NUMA, a processor can access its own local memory faster than non-local memory
+(memory local to another processor or memory shared between processors).
+Yarn Containers can make benefit of this NUMA design to get better performance by binding to a
+specific NUMA node and all subsequent memory allocations will be served by the same node,
+reducing remote memory accesses.
```

Comment:
Can we also mention that NUMA support for YARN Container has to be enabled only if worker node machines has NUMA support.